src/main/java/com/epam/tc/hw3/voids/IndexPage.java

```diff
@@ -0,0 +1,68 @@
+package com.epam.tc.hw3.voids;
+
+import com.epam.tc.hw3.AbstractBasePage;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class IndexPage extends AbstractBasePage {
```

Comment:
This page has bunch of the same elements with IndexFluentPage. Extend all common elements with come parent page.

Reply:
IndexPage and IndexFluentPage now have a common parent page.